posts/2020-11-26-MP-rest-client-Kubernetes-secrets-21001.adoc

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: "MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta - OpenLiberty.io
+seo-description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+blog_description: MP Rest Client 2.0, using Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MP Rest Client 2.0, using Kubernetes secrets in Liberty variables and more Jakarta EE 9 features in Open Liberty 21.0.0.1-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+
+Open Liberty 21.0.0.1-beta comes with MP Rest Client 2.0 (which is part of MicoProfile 4.0), this update adds support for HTTP procy servers and automatically following HTTP redirects. In 21.0.0.1-beta you can now use Kubernetes secrets in Liberty variables. Plus Jakarta Faces Container 3.0 and Jakarta Enterprise Beans Persistent Timer 4.0 included in Jakarta EE 9.
+
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<MP, MP Rest Client 2.0>>
+* <<kube, Using Kubernetes secrets in Liberty variables>>
+
+
+[#MP]
+=== MP Rest Client 2.0
+
+MicroProfile REST Client is a type-safe client API enabling rapid development of applications capable of consuming RESTful services. Version 2.0 is the latest update and add support for HTTP proxy servers, automatically following HTTP redirects, additional configuration options for JSON-B providers and how multiple query parameters should be specified, and support for Server Sent Events.
+
+To enable this feature, add `<feature>mpRestClient-2.0</feature>` to you list of features in the `<featureManager>` element - the `microprofile-4.0` convenience feature will also enable this feature.
```

Comment:
To enable this feature, add `<feature>mpRestClient-2.0</feature>` to you list of features in the `<featureManager>` element.  Alternatively, the `microprofile-4.0` convenience feature can be used to used instead.

Reply:
Alternatively, we should probably put these in code blocks to match how it's done below in the Jakarta features section

Reply:
Done